modules/nw-ovn-kubernetes-live-migration-about.adoc

```diff
@@ -0,0 +1,88 @@
+
+* Continuous service availability
+* Minimized downtime
+* Automatic node rebooting
+* Seamless transition from the OpenShift SDN network plugin to the OVN-Kubernetes network plugin
+
+Although a rollback procedure is provided, the live migration is intended to be a one-way process.
+
+include::snippets/sdn-deprecation-statement.adoc[]
+
+The following sections provide more information about the live migration method.
+
+[id="supported-platforms-live-migrating-ovn-kubernetes"]
+== Supported platforms when using the live migration method
+
+The following table provides information about the supported platforms for the live migration type.
+
+.Supported platforms for the live migration method
+[cols="1,1", options="header"]
+|===
+| Platform              | Live Migration
+
+| Bare metal hardware (IPI and UPI)             |&#10003;
+| Amazon Web Services (AWS) (IPI and UPI)       |&#10003;
+| Google Cloud Platform (GCP) (IPI and UPI)     |&#10003;
+| {ibm-cloud-name} (IPI and UPI)                |&#10003;
+| Microsoft Azure (IPI and UPI)                 |&#10003;
+| {rh-openstack-first} (IPI and UPI)            |&#10003;
+| VMware vSphere (IPI and UPI)                  |&#10003;
+| AliCloud (IPI and UPI)                        |&#10003;
+| Nutanix (IPI and UPI)                         |&#10003;
+|===
+
+[id="considerations-live-migrating-ovn-kubernetes-network-provider_{context}"]
+== Considerations for live migration to the OVN-Kubernetes network plugin
+
+Cluster administrators should consider the following before using the live migration method to the OVN-Kubernetes network plugin:
+
+* The live migration procedure is unsupported for clusters with OpenShift SDN multitenant mode enabled. 
+
+* Egress router pods block the live migration process. They must be removed before beginning the live migration process.
+
+* During the live migration, multicast, egress IP addresses, and egress firewalls are temporarily disabled. They can be migrated from OpenShift SDN to OVN-Kubernetes after the live migration process has finished.
+
+* Although a rollback procedure is provided, the migration is intended to be a one-way process. For users that want to rollback to OpenShift-SDN, migration from OpenShift-SDN to OVN-Kubernetes must have succeeded. Users can follow the same procedure below to migrate to the OpenShift SDN network plugin from the OVN-Kubernetes network plugin.
+
+* The live migration not supported on HyperShift clusters.
+
+* OpenShift SDN does not support IPsec. After migration, cluster administrators can enable IPsec.
+
+* OpenShift SDN does not support IPv6. Consequently, live migration to OVN-Kubernetes with IPv6 is not supported. Cluster administrators can enable dual-stack if necessary after migration is completed.
```

Comment:
Seems we do need this statement - `Consequently, live migration to OVN-Kubernetes with IPv6 is not supported`